### PR TITLE
Apply miscellaneous pre-v2.2.0 release fixes

### PR DESCRIPTION
**Pull request by PawelLipski**
Just what I've spotted while skimming through #1064

### CODE REVIEW

File: settings.gradle.kts [hunk 1]
```diff
@@ -1,11 +1,3 @@
-pluginManagement {
-  repositories {
-    gradlePluginPortal()
-    // For nl.littlerobots.version-catalog-update
```
Comment (PawelLipski) [inline settings.gradle.kts:4]: No longer needed, after all these reworks to how we refer to Gradle plugins

File: buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijVersionHelper.kt [hunk 2]
```diff
@@ -15,7 +14,7 @@ object IntellijVersionHelper {
     if (versionKey == null) {
       return emptyList()
     }
-    val regex = "/^[0-9].*$/".toRegex()
+    val regex = "^[0-9].*$".toRegex()
```
Comment (mkondratek) [inline buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijVersionHelper.kt:17]: Was it a bug?

File: buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/Spotless.kt [hunk 3]
```diff
@@ -37,9 +37,11 @@ fun Project.configureSpotless() {
   val isCI: Boolean by rootProject.extra
 
   if (!isCI) {
-    tasks.withType<AbstractCompile> { dependsOn("spotlessKotlinGradleApply") }
-    tasks.withType<JavaCompile> { dependsOn("spotlessJavaApply") }
-    tasks.withType<KotlinCompile> { dependsOn("spotlessKotlinApply") }
-    tasks.withType<ScalaCompile> { dependsOn("spotlessScalaApply") }
+    tasks {
```
Comment (mkondratek) [inline buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/Spotless.kt:40]: Nice

**Review by mkondratek — APPROVED** [review]

File: buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijVersionHelper.kt (hunk 2 above)
Comment (PawelLipski) [reply buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijVersionHelper.kt:17]: Yes... in Groovy `/.../` syntax is a regex, no quotes needed; nothing like that in Kotlin tho

**Review by MaciejG604 — APPROVED** [review]

**Pull request merged**